scripts/check_type_hint_coverage.py

```diff
@@ -3,7 +3,7 @@
 from typing import Dict, List, Optional
 
 TYPE_HINT_ERROR_THRESHOLD: int = (
-    2564  # This number is to be reduced as we annotate more functions!
+    2568  # This number is to be reduced as we annotate more functions!
```

Comment:
Looks like we are missing 4 type hints in this PR.

Reply:
@NathanFarmer I could not find them -- I do not think I added code without type hints...